examples/led_fade/config/config.exs

```diff
@@ -0,0 +1,9 @@
+use Mix.Config
+
+i2c_implementation =
+  case Mix.env() do
+    :prod -> ElixirALE.I2C
+    _ -> GrovePi.I2C
+  end
+
+config :grovepi, i2c: i2c_implementation
```

Comment:
I know that we do this in the alarm example, but I'm wondering if we need this. I don't think that the stubs help for the examples.